### ISSUE TITLE
D'oh. If you're seeing this, your DocumentRoot is set incorrectly. It should be set to the public directory. error from web server

### ISSUE THREAD

**Issue by rahulrgowda** [snipe/snipe-it#5507]
hi im getting D'oh. If you're seeing this, your DocumentRoot is set incorrectly. It should be set to the public directory. error when i host on my web server which is shared hostingsinpe-it works fine for me locally.please help me out

**Comment by snipe** [comment]
That page tells you what you need to do - you need to set the Document Root to public. If you're on shared hosting that doesn't allow you to change that, Snipe-IT may not work on that server.

**Comment by GeorgeVolak** [comment]
I too am getting this. I am unsure of where to point it to and how to express that. Below is an excerpt from my httpd.conf file: can you give some guidance on this? I've tried different things but haven't been able to get it to work.

# DocumentRoot: The directory out of which you will serve your
# documents. By default, all requests are taken from this directory, but
# symbolic links and aliases may be used to point to other locations.
#
DocumentRoot "${INSTALL_DIR}/www"
#DocumentRoot "c:\wamp\www\snipe-it\piblic\"
<Directory "${INSTALL_DIR}/www/">
    #
    # Possible values for the Options directive are "None", "All",
    # or any

**Comment by snipe** [comment]
@GeorgeVolak first off, your document root is commented out. Second off, you want to point it to the public directory - looks like you have a typo.

**Comment by GeorgeVolak** [comment]
Thank you Snipe. Sorry about that, actually I had it correct at first and it wasn't commented out but I still got the error. Do I need both lines for DocumentRoot? I found a reference to a file called httpd-vhosts.conf that was referenced in the WAMP tray control under Virtual Hosts. It showed the error DocumentRoot_error. This is where I found the reference to the httpd-vhosts.conf file. I found that file in the folder: C:\wamp\bin\apache\apache2.4.33\conf\extra

Below is a snip from that file:

 ServerAlias localhost
  DocumentRoot "${INSTALL_DIR}/www/snipe-it/public"
  <Directory "${INSTALL_DIR}/www/">
    Options +Indexes +Includes +FollowSymLinks +MultiViews

I edited the DocumentRoot line.  I found another version of this httpd-vhosts.conf file in the forder: C:\wamp\bin\apache\apache2.4.33\conf\original\extra. Not sure if I need to edit that one as it also has a reference to the DocumentRoot folder.

So now I have a green WAMP icon with no errors but still get the D'oh error.
